packages/ia-topnav/src/primary-nav.js

```diff
@@ -114,7 +114,10 @@ class PrimaryNav extends TrackedElement {
       <nav>
         <a class="link-home" href="https://${this.config.baseHost}" @click=${this.trackClick} data-event-click-tracking="${this.config.eventCategory}|NavHome">${icons.iaLogo}${logoWordmark}</a>
         ${this.searchMenu}
-        <a href="https://${this.config.baseHost}/upload/" class="upload">${icons.upload}Upload</a>
+        <a href="https://${this.config.baseHost}/upload/" class="upload">
+          ${icons.upload}
+          <span>Upload</span>
```

Comment:
QUESTION: Does putting these on their own lines introduce any undesired whitespace text nodes around the icon and text link?

Reply:
No, in fact one of the feedback items was that there was too little space between the icon and text. This allows me to not have to add padding to the child node.